src/Client/RestClient.php

```diff
@@ -78,11 +78,20 @@ public function __construct(
         // Create Guzzle client
         $guzzle = new Client($guzzleConfig);
 
+        // Use host from guzzle config if present (in previous versions of Guzzle, this was done automatically)
+        $defaultHostHeader = null;
+        foreach ($guzzleConfig['headers'] ?? [] as $name => $value) {
```

Comment:
Default hodnotu hlavicky vytiahneme z `guzzleConfig['headers']`, ... pretoze v novej verzii to Guzzle ignoruje (ostatne hlavicky takto funguju spravne, problem je iba s `Host`, ktore Guzzle teraz nastavuje uz z URL).